README.md

```diff
@@ -49,12 +50,14 @@ dependencies are required for native playback of audio files:
 * SndFile : [see here](http://www.mega-nerd.com/libsndfile/)
 * mpg123 : mpeg audio playback
 
-And for two examples require these dependencies to be built  
+Two of the packaged examples require the following dependencies to be built.  
 
 * PhysFS : alure-physfs
 * dumb : alure-dumb
 
-If you fail to meet the requirements for building the examples, compilation will simply be skipped. 
+If you fail to meet the requirements for building the examples compilation will simply be skipped. 
+This goes for all dependencies excluding OpenAL itself as well. For example, if any of the decoder libraries 
+aren't found, that given decoder will be skipped.
```

Comment:
This reads rather oddly. First mentioning the examples' requirements, then mentioning all requirements except for OpenAL itself too. It would probably be clearer and more succinct to simply say something along the lines if any dependency isn't found at build time, the related functionality will be disabled for the build.